### PR TITLE
feat(schema): add schema:migrate command

### PR DESCRIPTION
**Pull request by petrjasek**
it can detect schema updates and rebuild elastic indexes if needed

LBSD-897

### CODE REVIEW

**Comment by petrjasek** [comment]
@ancafarcas what do you think?
Idea would be that whenever you make changes which should trigger schema change, you would increment `SCHEMA_VERSION` in `superdesk.__init__.py`.
and that command should run when deploying changes.

**Comment by akintolga** [comment]
:+1: 

One question: It looks like this will re-index all the collections including text-archive. Is that right?

**Comment by ancafarcas** [comment]
@petrjasek I think it is a good idea. I would certainly adopt it.

**Comment by petrjasek** [comment]
@akintolga it depends on config, if you have it in separate index it won't touch it, only the main one

File: superdesk/commands/schema.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+from superdesk.lock import lock, unlock
+from superdesk.commands.rebuild_elastic_index import RebuildElasticIndex
+
+
+VERSION_ID = 'schema_version'
+
+
+def _get_version_db():
+    """Get db used for storing version information."""
+    return app.data.mongo.pymongo().db['superdesk']
```
Comment (actionless) [inline superdesk/commands/schema.py]: db name is hardcoded

Comment (petrjasek) [inline superdesk/commands/schema.py]: it's collection name not db

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**